api/src/main/java/com/datastrato/gravitino/authorization/SecurableObjects.java

```diff
@@ -195,4 +202,26 @@ static SecurableObject of(
     return new SecurableObjectImpl(
         metadataObject.parent(), metadataObject.name(), type, privileges);
   }
+
+  private static String getParentFullName(List<String> names) {
+    if (names.size() <= 1) {
+      return null;
+    }
+
+    return String.join(".", names.subList(0, names.size() - 1));
+  }
+
+  private static String getLastName(List<String> names) {
```

Comment:
Where is this method used?

Reply:
The enum `Name` type has `highBits` and `lowBits`, if `lowBits` more than 64, we can use `highBits`.

Reply:
OH, This is an invalid function. I rollback these codes.